types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts

```diff
@@ -49,14 +49,14 @@ configuration = {
 
 configuration = {
     // ...
-    plugins: [new MiniCssExtractPlugin()],
+    plugins: [new MiniCssExtractPlugin(), new MiniCssExtractPlugin({})],
```

Comment:
Not sure why an empty object param should be added here.

Reply:
Oh, it's just a test for coding issue. Allowed by package itself, all options are not required and comes with defualts.
Should I remove this? The contract allows user error (object with no properties is fully supported)
running that code is perfectly OK and creates:
```js
MiniCssExtractPlugin {
  options: {
    filename: '[name].css',
    ignoreOrder: false,
    chunkFilename: '[name].css'
  }
}
```

Reply:
I wasn't sure if it was a valid use case in practice and if not if it was worth testing?

I don't see a problem in keeping it though if you think it's important to have tested 👍